### PR TITLE
[bitnami/elasticsearch] Fix creation of serviceAccounts names into variables in _helpers.tpl

### PR DESCRIPTION
**Pull request by Hmr-ramzi**
**Description of the change**
Minor change to make creation of service accounts consistent with other charts like Kafka and Zookeeper by using variables inside _helpers.tpl instead of repeating code.

**Benefits**
No benefits just more consistent

**Possible drawbacks**
None

**Applicable issues**

  - None

**Additional information**

**Checklist** 
- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)
- [x] If the chart contains a `values-production.yaml` apart from `values.yaml`, ensure that you implement the changes in both files

:warning: Keep in mind that if you want to make changes to the kubeapps chart, please implement them in the [kubeapps repository](https://github.com/kubeapps/kubeapps/tree/master/chart/kubeapps). This is only a synchronized mirror.

### CODE REVIEW

File: bitnami/elasticsearch/templates/coordinating-deploy.yaml [hunk 1]
```diff
@@ -33,7 +33,7 @@ spec:
       tolerations: {{- include "elasticsearch.tplValue" (dict "value" .Values.coordinating.tolerations "context" $) | nindent 8 }}
       {{- end }}
       {{- if .Values.coordinating.serviceAccount.create }}
```
Comment (sameersbn) [inline bitnami/elasticsearch/templates/coordinating-deploy.yaml]: This **if** condition can be removed

File: bitnami/elasticsearch/templates/data-statefulset.yaml [hunk 2]
```diff
@@ -37,7 +37,7 @@ spec:
       tolerations: {{- include "elasticsearch.tplValue" (dict "value" .Values.data.tolerations "context" $) | nindent 8 }}
       {{- end }}
       {{- if .Values.data.serviceAccount.create }}
```
Comment (sameersbn) [inline bitnami/elasticsearch/templates/data-statefulset.yaml]: this one as well

File: bitnami/elasticsearch/templates/master-statefulset.yaml [hunk 3]
```diff
@@ -34,7 +34,7 @@ spec:
       tolerations: {{- include "elasticsearch.tplValue" (dict "value" .Values.master.tolerations "context" $) | nindent 8 }}
       {{- end }}
       {{- if .Values.master.serviceAccount.create }}
```
Comment (sameersbn) [inline bitnami/elasticsearch/templates/master-statefulset.yaml]: ...and this one too

**Comment by Hmr-ramzi** [comment]
Thanks for your request. Totally makes sense.

**Pull request merged**